CHANGELOG.md

```diff
@@ -18,7 +18,23 @@ and this project does not adheres to [Semantic Versioning](http://semver.org/).
 
 ## [0.1] - 2014-11-15
 ### Added
-- First version of mex-wholebodymodel, compatible with wholebodyinterface v0.1 . 
+- First version of mex-wholebodymodel, compatible with wholebodyinterface v0.1. 
+
+## [0.2] - 2017-01-25
+### Added
+- New version of mex-wholebodymodel. A complete refactory of the toolbox has been done.
+
+### Renamed
+- WBM-wrappers functions have been renamed, and some of the outputs are listed in a different way.
+
+### Moved
+- WBM-based controllers have been refactored and moved on a different branch, called `WBM-Controllers`; 
+
+### Removed and merged 
+- Utility functions have been merged with WMB-class/utilities. Unnecessary or repeated functions have been removed.
+
+### New visualizer
+- New toolbox for visualize robot simulation bades on iDyntree library. For further info, see README.
```

Comment:
ok, I will correct this typo after merging, in order to modify the CHANGELOG in a single commit